build.gradle

```diff
@@ -1,17 +1,20 @@
 buildscript {
   repositories {
     mavenCentral()
-    maven { url "https://jitpack.io" }
+    maven { url 'https://jitpack.io' }
+    maven { url 'https://plugins.gradle.org/m2/' }
```

Comment:
Latest version of spotbugs for gradle is available in `https://plugins.gradle.org/m2/`